### PR TITLE
TRL-007 Archive_verification

### PR DESCRIPTION
**Pull request by helenver**
[allure.zip](https://github.com/helenver/TRL-Trello/files/11649713/allure.zip) was created on 04.06
[allure.zip](https://github.com/helenver/TRL-Trello/files/11667994/allure.zip) was updated on 06.06

**Scope:**
_Preconditions:_

- Board creation
- List creation
- Cards creation

_Checks:_

- List archive
- List dearchive 
- All cards archive 
- All cards dearchive
- One card archive 
- One card dearchive
- Archived card deletion
- Board deletion

### CODE REVIEW

File: src/main/resources/story/regression/archive_checks.story [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Scenario: Log in as a board creator
+When I log in with ${email} and ${password}
+
+Scenario: Open the board
+When I click on element located by `By.xpath(//div[@title='${boardName}'])`
+
+Scenario: Archive the list via List DDL
+When I click on element located by `By.xpath(//*[@aria-label="${listName}"]/../div[@class="list-header-extras"])`
+When I click on element located by `By.xpath(//a[text()="Archive this list"])`
+Then number of elements found by `By.xpath(//*[@aria-label="Iapetus"]//ancestor-or-self::*[@class="list js-list-content"])` is equal to `0`
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/archive_checks.story:12]: This xpath is not descriptive enough

File: src/main/resources/story/regression/archive_checks.story [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Scenario: Find the archived list in the Archive
+When I click on element located by `By.xpath(//button[@aria-label="Show menu"]//span[@data-testid="OverflowMenuHorizontalIcon"])`
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/archive_checks.story:15]: Please shorten xpath, use only descriptive elements where applicable

File: src/main/resources/story/regression/archive_checks.story [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+When I click on element located by `By.xpath(//span[text()="${secondCardName}"])`
+When I click on element located by `By.xpath(//span[text()='Archive'])`
+When I click on element located by `By.xpath(//a[@aria-label="Close dialog"])`
+When I click on element located by `By.xpath(//button[@aria-label="Show menu"]//span[@data-testid="OverflowMenuHorizontalIcon"])`
+When I click on element located by `By.xpath(//a[@class="js-delete"])`
+When I click on element located by `By.xpath(//input[@value="Delete"])`
+Then number of elements found by `By.xpath(//span[text()="${secondCardName}"])` is equal to `0`
+
+Scenario: Delete board via API
+When I delete the board with ${boardId} via API
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/archive_checks.story:87]: Please add a newline

Comment (helenver) [reply src/main/resources/story/regression/archive_checks.story:87]: Done ✅

File: src/main/resources/story/regression/archive_checks.story (hunk 2 above)
Comment (helenver) [reply src/main/resources/story/regression/archive_checks.story:15]: Done ✅

File: src/main/resources/story/regression/archive_checks.story (hunk 1 above)
Comment (helenver) [reply src/main/resources/story/regression/archive_checks.story:12]: Corrected : //*[@aria-label="${listName}"]

File: src/main/resources/story/regression/archive_checks.story [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+When I log in with ${email} and ${password}
+
+Scenario: Open the board
+When I click on element located by `By.xpath(//div[@title='${boardName}'])`
+
+Scenario: Archive the list via List DDL
+When I click on element located by `By.xpath(//*[@aria-label="${listName}"]/../div[@class="list-header-extras"])`
+When I click on element located by `By.xpath(//a[text()="Archive this list"])`
+Then number of elements found by `By.xpath(//*[@aria-label="${listName}"])` is equal to `0`
+
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/archive_checks.story:13]: Remove extra line

**Review by ruslan-tyshchanka — APPROVED** [review]

File: src/main/resources/story/regression/archive_checks.story (hunk 4 above)
Comment (helenver) [reply src/main/resources/story/regression/archive_checks.story:13]: Done ✅

**Pull request merged**